ios/example-pip-app/src/Info.plist

```diff
@@ -51,6 +51,8 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
+	<false/>
```

Comment:
There's a component being used in JitsiMeet SDK that disables the phone status bar, but for it to work you need to enable that flag in the consumer app. I don't think we need to comment on that because the logs will constantly tell you to enable it and also every now and then the RN red window screen will appear to indicate you have to do it.